authentication/managing_cloud_provider_credentials/cco-mode-mint.adoc

```diff
@@ -17,7 +17,7 @@ The CCO uses the `admin` credential to process the `CredentialsRequest` objects
 With mint mode, each cluster component has only the specific permissions it requires.
 Cloud credential reconciliation is automatic and continuous so that components can perform actions that require additional credentials or permissions.
 
-For example, a minor version cluster update (such as updating from {product-title} 4.16 to 4.17) might include an updated `CredentialsRequest` resource for a cluster component.
+For example, a minor version cluster update (such as updating from {product-title} {ocp-nminus1} to {product-version}) might include an updated `CredentialsRequest` resource for a cluster component.
```

Comment:
:nerd_face: AWESOME :tada:

How far back do these attributes exist? I would like to backport them :joy: